app/assets/javascripts/profile_table.js

```diff
@@ -0,0 +1,77 @@
+var ProfileTable = (function(){
+  var widget, selector;
+
+  ProfileTable.prototype = {
+    append: function(){
+      widget = $(selector).editableTableWidget();
+      widget.on('change', parseTableToJSON.bind(this));
+
+      addClickListenersToAddRow();
+      addClickListenersToDeleteRow();
+      parseTableToJSON();
+    }
+  };
+
+  function parseTableToJSON(){
+    $("#testing_ground_technology_profile").text(tableToProfile());
+  };
+
+  function tableToProfile(){
+    var technologyProfile = _.map(tableRows(), function(tableRow){
+      return rowToTechnologyObject(tableRow);
+    });
+    return JSON.stringify(_.groupBy(technologyProfile, 'node'));
```

Comment:
Is Underscore really needed? It's a big dependency for only three uses. `_.groupBy` is pretty simple, and `Array.prototype.map` [is well-supported now](https://kangax.github.io/compat-table/es5/#Array.prototype.map), so the two instances of `_.map` can probably be replaced.

```javascript
function groupBy(elements, key) {
    var grouped = {},
        length  = elements.length,
        i, element;

    for (i = 0; i < length; i++) {
        element = elements[i];

        if (! grouped.hasOwnProperty(element[key])) {
            grouped[element[key]] = [];
        }

        grouped[element[key]].push(element);
    }

    return grouped;
}

groupBy([
    { category: 'main', name: 'First' },
    { category: 'rest', name: 'Second' },
    { category: 'main', name: 'Third' },
    { category: 'rest', name: 'Fourth' }
], 'category');
```